var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -41,8 +41,9 @@ class Legion(CMakePackage):
     depends_on('mpi', when='conduit=mpi')
     depends_on('cuda@10.0:11.9', when='+cuda_unsupported_compiler')
     depends_on('cuda@10.0:11.9', when='+cuda')
+    depends_on('hip@4.1.0', when='+hip')
```

Comment:
```suggestion
    depends_on('hip@4.1.0:', when='+hip')
```
Just 4.1.0 or 4.1.0 and above? In the latter case. Also, have you seen https://spack.readthedocs.io/en/latest/build_systems/rocmpackage.html?

Reply:
Ah!  Thanks!  Nope, I missed that.   Will take a look -- I'd like to get this minimal update out the door and then make more changes for both cuda and rocm (that will get some users running sooner rather than later).  Thanks again!